extensions/adobe/experience/decisioning/personalized-content-option.schema.json

```diff
@@ -26,7 +26,17 @@
   "description": "Personalized decision options form the set of choices for any decision actvity. The objective for decisioning is to take a large inventory of items and apply numerous constraint rules to that inventory to narrow it down and then to rank the qualifying options according to a criteria. The resulting propositions assemble and personalize the experience for specific individuals.",
   "definitions": {
     "personalized-content-option": {
-      "properties": {}
+      "properties": {
+        "xdm:fCapConstraintsLastIndex": {
```

Comment:
@jshaik-adobe , just wanted to check if we have considered adding this field as a property in frequency-capping-constraints itself.

https://github.com/adobe/xdm/blob/01c2aafbc1ff30165f3f6a557928c2e53c6c7717/extensions/adobe/experience/decisioning/frequency-capping-constraints.schema.json#L20

Reply:
thanks @usankara , I updated as per review comments